hawkular-alerts-engine-extensions/hawkular-alerts-aggregated-events-alerter/src/main/java/org/hawkular/alerts/extensions/AggregatedEventsExtension.java

```diff
@@ -42,21 +42,29 @@
 import org.hawkular.alerts.api.model.trigger.Trigger;
 import org.hawkular.alerts.api.services.DefinitionsEvent;
 import org.hawkular.alerts.api.services.DefinitionsService;
-import org.hawkular.alerts.api.services.ExtensionEventListener;
+import org.hawkular.alerts.api.services.EventExtension;
 import org.hawkular.alerts.api.services.ExtensionsService;
 import org.hawkular.alerts.api.services.PropertiesService;
 import org.jboss.logging.Logger;
 
 /**
- * Manages the APM expression evaluations and interacts with the Alerts system.
+ * This EventExtension is responsible of the following tasks:
+ *
+ * - It register a DefinitionsListener into the Alerting engine.
+ *   It will process ExternalConditions tagged with HawkularExtension=AggregatedEvents into CEP rules.
```

Comment:
Very minor and totally optional: AggregatedEvents -> EventAggregation

Just a change from using past tense to something more active.  This may make it easier to talk about, "If enabled, incoming events are first processed by the EventAggregation extension..."